### DISCUSSION TITLE
Feature proposal: struct-like formatting when doing table-like tests

### DISCUSSION THREAD

**Discussion by michelececcacci** [mvdan/gofumpt#259] [Ideas]
Since this is already supported for struct definition, why not add it for structs when working with tables?
For example, it  would turn something like this:
```go
	cases := []struct {
		s string
		t string
	}{
		{s: "short", t: "anotherstring"},
		{s: "reallylongstring", t: "somestring"   },
	}
```
into something like this:

```go
	cases := []struct {
		s string
		t string
	}{
		{s: "short",            t: "anotherstring"},
		{s: "reallylongstring", t: "somestring"   },
	}
```

**Comment by mvdan** [comment]
Unfortunately we can't do this, as gofmt would fight us here - gofumpt would want to add the whitespace, and gofmt would want to remove it. See the README for the design principles. In other words, if you want this alignment, gofmt has to add it.

**Reply by michelececcacci** [reply]
Oh, ok. Sorry! Thanks for the answer.
